main/define/test/src/mill/define/MacroErrorTests.scala

```diff
@@ -94,8 +94,9 @@ object MacroErrorTests extends TestSuite {
             }
           }
         """)
+        pprint.log(e)
         assert(e.msg.contains(
-          "Modules and Tasks can only be defined within a mill Module"
+          "Task{} members must be defs defined in a Cacher class/trait/object body"
```

Comment:
Since `Cacher` never surfaces in our docs (and API), we probably should list the most likely candidate `Module`.

Reply:
```suggestion
          "Task{} members must be defs defined in a Cacher class/trait/object body like Module"
```

Reply:
This actually comes from the mill-moduledefs repo, so we'll need to fix it upstream there https://github.com/com-lihaoyi/mill-moduledefs/blob/33a27d470a8f27a1cb2ad9e71d8c25b24294eee8/moduledefs/src-2/Cacher.scala#L29

Reply:
https://github.com/com-lihaoyi/mill-moduledefs/pull/20